content/zh-cn/docs/reference/ports-and-protocols.md

```diff
@@ -15,7 +15,7 @@ weight: 50
 When running Kubernetes in an environment with strict network boundaries, such 
 as on-premises datacenter with physical network firewalls or Virtual 
 Networks in Public Cloud, it is useful to be aware of the ports and protocols 
-used by Kubernetes components
+used by Kubernetes components.
 -->
```

Comment:
It's the only change in this PR, I don't think it will add any value to k/website.

Reply:
The document should be the same, I think so
example
https://github.com/kubernetes/website/pull/34800/files
https://github.com/kubernetes/website/pull/34379/files

Reply:
This change I think is too small to make sense and wastes reviewers' time. I would like to avoid such PRs as much as possible.

Reply:
Ok

Reply:
Thank you very much for the catch as well~ 
We can update this file together the next time we find changes to it.